src/Components/OATHeader/OATHeader.tsx

```diff
@@ -129,16 +130,20 @@ const OATHeader = ({ elements, dispatch, state }: OATHeaderProps) => {
                     const validJson = await parseModel(content);
                     if (!validJson) {
                         items.push(newItem.content);
+                    } else {
+                        allValidFiles = false;
```

Comment:
We can break out of the loop at this point I guess?

Reply:
Actually I was asked to look for all the errors 
![image](https://user-images.githubusercontent.com/97867829/172205956-c519990c-de3d-4d6f-ac34-bc4064ccfe68.png)
If I break the loop only the first one will be shown